app.js

```diff
@@ -68,4 +68,33 @@ let sum1 = 0
 for (let i = 11; i <= 30; i += 2) {
 	sum1 += i;
 }
-console.log(sum1);
\ No newline at end of file
+console.log(sum1);
+
+console.log("==============Challenge 8=================");
+// Coding challenge #8: Create a function that will convert from Celsius to Fahrenheit
+
+let celcius = 0
+
+function celciusToFahrenheit(celcius) {
+	return (celcius * 9 / 5) + 32;
+}
+
+console.log("==============Challenge 9=================");
+// Coding challenge #9: Create a function that will convert from Fahrenheit to Celsius
+
+
+function fahrenheitToCelcius(fahrenheit) {
+	return (fahrenheit - 32) * 5 / 9;
+}
+
+console.log("==============Challenge 10=================");
+
+// Coding challenge #10: Calculate the sum of numbers in an array of numbers
+let sum3 = 0
```

Comment:
You are creating a function that does everything encapsulated (Good), but relies on a  global variable (Bad).  This belongs inside the function, and every time you name a variable "name + random number" you're doing it wrong.

By placing it inside the function, you'll be ablet o simply call it "sum".

Reply:
I had changed this to be inside the function, realising my mistake. Removed all the sum2 sum3 etc